src/app/actions.ts

```diff
@@ -0,0 +1,13 @@
+'use server'
+
+import { createItem } from '@/lib/models/item'
+import { getCurrentAuthUser } from '@/lib/models/user'
+import { revalidatePath } from 'next/cache'
+
+export const addItemAction = async (formData: FormData): Promise<void> => {
+  const authUser = getCurrentAuthUser()
+  const name = formData.get('name') as string
+  const description = formData.get('description') as string
+  await createItem(authUser, name, description)
```

Comment:
I want my core models to be as agnostic as possible to dependencies and tooling. At this interface, I get the user from Clerk and I also break the html FormData type into its string constituents so that those don't get into my core business logic.